src/app/components/Disclaimer/index.tsx

```diff
@@ -0,0 +1,78 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { useContext, PropsWithChildren } from 'react';
+import useToggle from '#hooks/useToggle';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import { GridItemLarge } from '#components/Grid';
```

Comment:
While we're in here, could we update the imports to use a relative path instead of the # alias, as per the React Component Standards?